src/metric_voting/elections.py

```diff
@@ -595,13 +773,35 @@ class Monroe(Election):
     "Achieving Fully Proportional Representation: Approximability Results"
     Skowron et al (2013)
     (https://arxiv.org/abs/1312.4026)
+    NOTE: See section 4.7 starting on page 32.
     
     "What Do Multiwinner Voting Rules Do? An Experiment Over the Two-Dimensional Euclidean Domain"
     Elkind et al (2019)
     (https://arxiv.org/abs/1901.09217)
     
-    NOTE: As far as I can tell, these solvers output deterministic answers,
-    even in cases where there are multiple optimal solutions...Not sure how to handle this.
+    

```

Comment:
You should add the disclaimer in this
```suggestion
    NOTE: This is a basic implementation of the linear program and it will not randomly 
    break ties between optimal solutions. The LP will choose the solution deterministically 
    using its own tiebreaking mechanisms. See MonroeTiebreak for controlled randomization.
```